share/jupyter/voila/templates/base/page.html

```diff
@@ -20,7 +20,7 @@
 
   </head>
 
-  <body data-base-url="{{ base_url }}voila/">
+  <body data-base-url="{{ base_url }}voila/" data-in-voila>
```

Comment:
I propose to use the same naming schema as in Notebook


https://github.com/jupyter/notebook/blob/3b8e6736075279f80cd1e590c71a872dc5ad2c11/notebook/templates/notebooks.html#L15

```suggestion
  <body data-base-url="{{ base_url }}voila/" data-voila="voila">
```